.github/workflows/storybook-chromatic.yml

```diff
@@ -1,6 +1,10 @@
 name: QA
 
-on: [pull_request]
+on:
+  push:
+    branches-ignore:
+      # Matches release branches
+      - "[0-9]+.[0-9]+"
```

Comment:
AFAIK it will trigger the CI on tag push as well. Is this behavior wanted or not? If not, then `tags-ignore: *` might be needed https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore

Reply:
Good catch - I will fix it 🙇🏻